app/views/finders/_filter_button.html.erb

```diff
@@ -1,4 +1,5 @@
-<button class="app-c-button-as-link app-mobile-filters-link js-show-mobile-filters"
+<div style="position:relative;">
```

Comment:
This should be a class rather than an inline style - it's better practice and will soon be outlawed under our CSP.

Reply:
Whoops, good spot, was meant to be temporary for testing!